engine/move_mon.asm

```diff
@@ -201,24 +223,37 @@ endr
 rept 4
 	inc de
 endr
-	ld a, 70
+
+	; Initialize happiness.
+	ld a, BASE_HAPPINESS
 	ld [de], a
 	inc de
+
 	xor a
+	; PokerusStatus
 	ld [de], a
 	inc de
+	; CaughtData/CaughtTime/CaughtLevel
 	ld [de], a
 	inc de
+	; CaughtGender/CaughtLocation
 	ld [de], a
 	inc de
+
+	; Initialize level.
 	ld a, [wCurPartyLevel]
 	ld [de], a
 	inc de
+
 	xor a
+	; Status
 	ld [de], a
 	inc de
+	; Unused
 	ld [de], a
 	inc de
+
+	; Initialize HP.
 	ld bc, 10
```

Comment:
This ``ld bc, 10`` can probably use some constant? (not sure if there are more of these hardcoded values around, but it would be neat to fix it/them like you've done with a few others)

Reply:
I'm having trouble tracing the origins and current value of `hl`, but if I were to know that it'd not be much more than `structure_target - structure_base`.